packages/api/orm/migration/1638312144983-initial.ts

```diff
@@ -0,0 +1,128 @@
+        contributors character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        "readOnlyOperators" character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        name character varying(100) NOT NULL,
+        description character varying(300),
+        "portfolioId" uuid,
+        CONSTRAINT "fk_application_portfolio" FOREIGN KEY ("portfolioId") REFERENCES portfolio(id)
+    );
+    
+    ALTER TABLE application OWNER TO atat_api_admin;
+    
+    CREATE TABLE clin (
+        id uuid DEFAULT uuid_generate_v4() CONSTRAINT "pk_clin" PRIMARY KEY,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "clinNumber" character varying(4) NOT NULL,
+        "idiqClin" character varying NOT NULL,
+        "totalClinValue" money NOT NULL,
+        "obligatedFunds" money NOT NULL,
+        "popStartDate" date NOT NULL,
+        "popEndDate" date NOT NULL,
+        "taskOrderId" uuid,
+        CONSTRAINT "fk_clin_task_order" FOREIGN KEY ("taskOrderId") REFERENCES task_order(id)
+    );
+    
+    ALTER TABLE clin OWNER TO atat_api_admin;
+    
+    CREATE TABLE environment (
+        id uuid DEFAULT uuid_generate_v4() CONSTRAINT "pk_environment" PRIMARY KEY,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "provisioningStatus" provisioning_status_enum DEFAULT 'PENDING'::provisioning_status_enum NOT NULL,
+        administrators character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        contributors character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        "readOnlyOperators" character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        name character varying(100) NOT NULL,
+        "applicationId" uuid,
+        CONSTRAINT "fk_environment_application" FOREIGN KEY ("applicationId") REFERENCES application(id)
+    );
+    
+    ALTER TABLE environment OWNER TO atat_api_admin;
+    
+    CREATE TABLE portfolio (
+        id uuid DEFAULT uuid_generate_v4() CONSTRAINT "pk_portfolio" PRIMARY KEY,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "provisioningStatus" provisioning_status_enum DEFAULT 'PENDING'::provisioning_status_enum NOT NULL,
+        administrators character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        contributors character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        "readOnlyOperators" character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        name character varying(100) NOT NULL,
+        description character varying(300),
+        owner character varying NOT NULL,
```

Comment:
```suggestion
        owner character varying NOT NULL,
```
Owner is a User with an email, first name, and last name
```
  User:
      required:
      - email
      - firstName
      - lastName
      properties:
        email:
          type: string
          format: email
        firstName:
          type: string
        lastName:
          type: string
```